docs/src/main/asciidoc/spring-integration-pubsub.adoc

```diff
@@ -309,10 +309,10 @@ The first patterns have precedence over the following ones.
 In the previous example, the `"*"` pattern means every header is mapped.
 However, because it comes last in the list, https://docs.spring.io/spring-integration/api/org/springframework/integration/util/PatternMatchUtils.html#smartMatch-java.lang.String-java.lang.String...-[the previous patterns take precedence].
 
-=== Sample
+==== Samples
```

Comment:
Can you show a screenshot comparing before and after?

Reply:
Before:
![image](https://user-images.githubusercontent.com/3209274/81018170-fa2cc080-8e31-11ea-9f19-1dabe9b7c8d7.png)

After:
![image](https://user-images.githubusercontent.com/3209274/81018188-06b11900-8e32-11ea-9ec5-7811a5e42831.png)

I prefer the After here because it is more clear what the sample is a sample of rather than the sample appearing at a equivalent level to the other top-level sections inside spring-integration.